docs/_templates/ethicalads.html

```diff
@@ -0,0 +1,4 @@
+<div class="rtd-sponsorship">
+  <div class="raised" data-ea-publisher="readthedocs" data-ea-type="text"></div>
```

Comment:
I wonder if we should have some content in here for when ads don't load/show (eg. for Gold users?) so they aren't confused by the space.

Reply:
This is actually trickier than I thought. The goal of pre-allocating the space is so we don't get the page re-flowing after the fact. However, there isn't currently a way to hook into the response from the sustainability data API.